BaragonService/src/main/java/com/hubspot/baragon/service/elb/ApplicationLoadBalancer.java

```diff
@@ -112,21 +114,21 @@ public void syncAll(Collection<BaragonGroup> baragonGroups) {
 
             if (configuration.isPresent() && configuration.get().isDeregisterEnabled()) {
               LOG.debug("De-registering old instances for group {}", baragonGroup.getName());
-              deregisterIfOld(baragonGroup, targetGroup, baragonAgents, targets);
+              deregisterRemovableTargets(baragonGroup, targetGroup, baragonAgents, targets);
             }
           } else {
-            LOG.debug("Not TargetGroup for Baragon Group {}", baragonGroup);
+            LOG.debug("No TargetGroup for Baragon Group {}", baragonGroup);
           }
 
           LOG.debug("ELB sync complete for group {}", baragonGroup);
+        } catch (AmazonClientException acexn) {
+          LOG.error("Could not retrieve elb information due to ELB client error {}", acexn);
+          exceptionNotifier.notify(acexn, ImmutableMap.of("groups", baragonGroups == null ? "" : baragonGroups.toString()));
```

Comment:
nit: We're already accessing `baragonGroups` above by entering the for loop, probably don't need an additional null check